src/main/java/com/softserve/academy/spaced/repetition/repository/CategoryRepository.java

```diff
@@ -19,4 +18,7 @@
 
     @Query(value = "SELECT c.name, c.description, c.image FROM Category c WHERE c.id = :categoryId")
     List<Category> hasAccessToCategory(@Param("categoryId") Long categoryId);
+
+    @Query(value = "SELECT c FROM Category c WHERE c.name LIKE %:searchString% OR c.description LIKE %:searchString%")
```

Comment:
You may want to get rid of the @Query - https://stackoverflow.com/a/25363398
See Containing at https://docs.spring.io/spring-data/jpa/docs/current/reference/html/

Reply:
See also the comment to https://stackoverflow.com/a/25363398

> It may be useful also to ignore case for a query, using: findByPlaceIgnoreCaseContaining(String place);

Reply:
Done!